account_banking_payment_transfer/model/account_payment.py

```diff
@@ -253,3 +268,22 @@ def action_sent(self):
         # State field is written by act_sent_wait
         self.write({'date_sent': fields.Date.context_today(self)})
         return True
+
+    @api.multi
+    def partial(self):
+        self.ensure_one()
+        view_id = self.env.ref('account.view_move_line_tree').id
```

Comment:
This issue has been fixed in Odoo at this revision: https://github.com/odoo/odoo/commit/aaf9badbceaf683603b4667dd3d29e2784e78e6f
Then, I delete my comment which is outdated.